src/service/NylasService/NylasService.ts

```diff
@@ -157,6 +157,25 @@ export class NylasService extends APIClient {
         return await this.invokeApi('/config', 'PATCH', config);
     }
 
+    /**
+     * Sets user default email
+     * @param email
+     */
+    async setDefaultEmail(email: string): Promise<AxiosResponse<any>> {
+        return await this.invokeApiWithErrorHandling('/default-account', 'POST', undefined, {
+            queryParams: {
+                email: email,
+            },
+        });
+    }
```

Comment:
- async methods should not have a get / set prefix
- invokeApiWithErrorHandling do have their own response type

```suggestion
    async updateDefaultEmail(email: string) {
        return await this.invokeApiWithErrorHandling<TYPE>('/default-account', 'POST', undefined, {
            queryParams: {
                email: email,
            },
        });
    }
```

Reply:
done